### ISSUE TITLE
Does not support import.meta.ROLLUP_FILE_URL_referenceId

### ISSUE THREAD

**Issue by CodeWitchBella** [PepsRyuu/nollup#30]
With following plugin:

```js
const fs = require('fs')
const { promisify } = require('util')
const path = require('path')

function url({ exts = ['.svg', '.png', '.jpg', '.gif'] } = {}) {
  return {
    name: 'rollup-plugin-custom-url',
    async load(id) {
      if (exts.some(ext => id.endsWith(ext))) {
        const ref = this.emitFile({
          type: 'asset',
          source: await promisify(fs.readFile)(id, { encoding: 'binary' }),
          name: path.basename(id),
        })
        return 'export default import.meta.ROLLUP_FILE_URL_' + ref
      }
      return null
    },
  }
}
```

And code importing for example .png file:

```js
import url from './logo.svg'
const el = document.createElement('img')
el.src = url
document.body.appendChild(el)
```

I get following error
```js
{ SyntaxError: Unexpected token (1:21)
    at Parser.pp$4.raise (<project>/node_modules/acorn/dist/acorn.js:2836:15)
    at Parser.pp.unexpected (<project>/node_modules/acorn/dist/acorn.js:689:10)
    at Parser.pp$3.parseExprImport (<project>/node_modules/acorn/dist/acorn.js:2271:12)
    at Parser.pp$3.parseExprAtom (<project>/node_modules/acorn/dist/acorn.js:2254:21)
    at Parser.pp$3.parseExprSubscripts (<project>/node_modules/acorn/dist/acorn.js:2089:21)
    at Parser.pp$3.parseMaybeUnary (<project>/node_modules/acorn/dist/acorn.js:2066:19)
    at Parser.pp$3.parseExprOps (<project>/node_modules/acorn/dist/acorn.js:2010:21)
    at Parser.pp$3.parseMaybeConditional (<project>/node_modules/acorn/dist/acorn.js:1993:21)
    at Parser.pp$3.parseMaybeAssign (<project>/node_modules/acorn/dist/acorn.js:1968:21)
    at Parser.pp$1.parseExport (<project>/node_modules/acorn/dist/acorn.js:1431:33) pos: 21, loc: Position { line: 1, column: 21 }, raisedAt: 22 }
ParseError: src/img/logo.svg
Error: SyntaxError: Unexpected token (1:21)
    export default import.meta.ROLLUP_FILE_URL_logo_svg
                        ^
    at parse (<project>/node_modules/nollup/lib/index.js:93:23)
```

This was mostly experiment in writing simple replacement for [rollup-plugin-url](https://github.com/rollup/rollup-plugin-url)

**Comment by PepsRyuu** [comment]
Thanks for reporting the issue! I haven't gotten around previously to implementing that particular API feature. I think I have a basic version of it working though on a branch: https://github.com/PepsRyuu/nollup/commit/1410dfc8a76692cf06051d301c10874dce40ac61

Will need to do more testing and check for other possible variations, but should have something for this API feature soon.

**Comment by CodeWitchBella** [comment]
That commit works for my project :slightly_smiling_face:

**Comment by PepsRyuu** [comment]
Released in ```0.7.3``` 🙂

**Comment by CodeWitchBella** [comment]
Awesome :tada:
